apps/desktop/src/main/terminal-host/pty-subprocess.ts

```
@@ -92,6 +101,82 @@ function sendError(message: string): void {
+		altEnter:
+			data.includes("\x1b[?1049h") ||
+			data.includes("\x1b[?1047h") ||
+			data.includes("\x1b[?47h"),
+		altLeave:
+			data.includes("\x1b[?1049l") ||
+			data.includes("\x1b[?1047l") ||
+			data.includes("\x1b[?47l"),
+		clearBelow: data.includes("\x1b[J"),
+		clearScreen: data.includes("\x1b[2J"),
+		clearScrollback: data.includes("\x1b[3J"),
+		scrollRegion: /\x1b\[[0-9;]*r/.test(data),
```

Biome (2.4.11)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)